src/components/picker/types.ts

```diff
@@ -252,7 +252,8 @@ export type PickerPropsWithMulti = PickerBaseProps & {
   onChange?: (value: PickerMultiValue) => void;
 };
 
-export type PickerProps = PickerPropsWithSingle | PickerPropsWithMulti;
+// Using Pick to ensure TypeScript recognizes the migrate prop from PickerPropsDeprecation
+export type PickerProps = (PickerPropsWithSingle | PickerPropsWithMulti) & Pick<PickerPropsDeprecation, 'migrate'>;
 
```

Comment:
PickerProps extends PickerBaseProps which extends PickerPropsDeprecation.
we don't need this.